main.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
This `no newline at the end of file` warning that GitHub is showing can also be fixed with Prettier.